lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DBOFavoriteDAOImplAutowiredTest.java

```diff
@@ -262,24 +262,33 @@ public void testGetFavoritesEntityHeader() throws Exception {
 		favoritesToDelete.add(fav1created);
 		Favorite fav2created = favoriteDao.add(fav2);
 		favoritesToDelete.add(fav2created);
-		
+
 		PaginatedResults<EntityHeader> favs = favoriteDao.getFavoritesEntityHeader(creatorUserGroupId.toString(), Integer.MAX_VALUE, 0);
 
 		assertEquals(2, favs.getTotalNumberOfResults());		
 		assertEquals(2, favs.getResults().size());
 		
-		EntityHeader eh1 = null;
-		EntityHeader eh2 = null;
-		for(EntityHeader eh : favs.getResults()) {
-			if(eh.getName().equals(node1Name)) eh1 = eh;
-			if(eh.getName().equals(node2Name)) eh2 = eh;
-		}
+		EntityHeader eh1 = favs.getResults().get(0);
+		EntityHeader eh2 = favs.getResults().get(1);

```

Comment:
I added an `ORDER BY id` so the order results are returned should be deterministic, based on entity ID. In this test, the entities are created synchronously, so this order should always hold.

Reply:
I suggest adding a test in which pagination is not trival/default, e.g., make offset=1 and page size=1 and make sure you get exactly one page back.

Reply:
I think I capture what you're talking about in lines 288-291. If that's not what you're looking for, I need some further clarification

Reply:
Yes, you are right.  My mistake, I didn't see those two tests.